lib/glific/messages.ex

```diff
@@ -244,6 +244,23 @@ defmodule Glific.Messages do
     end
   end
 
+  @doc false
+  @spec create_and_send_message_to_contacts(map(), []) :: {:ok, Message.t()}
+  def create_and_send_message_to_contacts(message, contact_ids) do
+    contact_ids
+    |> Enum.reduce(fn contact_id, _ ->
+      contact_attrs = %{
+        receiver_id: contact_id,
+        contact_id: contact_id
+      }
+
+      message = Map.merge(message, contact_attrs)
+
+      with {:ok, message} <- create_message(message),
```

Comment:
Do you think we should remove the code duplicity and use create_and_send_message function?

Reply:
Yes, will use the create_and_send_message function.